scripts/download_json/index.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+/**
+ * Random scripts
+ * @module Scripts
+ */
+
+/**
+ * Generates the string to use as download link of a object in JSON format
+ *
+ *
+ * @param {object} Object - Object that will be downloaded as JSON
+ * Example:
+ *  <a href={getJSONDownloadString(object)} download={`${fileName}.json`}>
+ *    Download JSON
+ *  </a>
+ * @returns {string}
+ */
+const getJSONDownloadString = (object) => {
```

Comment:
I think we should change the name of the parameter received to obj. I know that `object` is not a reserved word but since `Object` is a default constructor in js, and `typeof {}` returns `"object"`, we might want to avoid the word.
We can change it to `obj`